### PR TITLE
add Important hint at getting-start.md

### PR DESCRIPTION
**Pull request by denghuilu**

### CODE REVIEW

File: doc/getting-started.md [hunk 1]
```diff
@@ -370,3 +371,10 @@ print(water.get_positions())
 ```
 [DP]:https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.120.143001
 [DP-SE]:https://dl.acm.org/doi/10.5555/3327345.3327356
+
+## Important hint
```
Comment (amcadmus) [inline doc/getting-started.md:375]: I would rename the title as "Know limitations"

File: doc/getting-started.md [hunk 2]
```diff
@@ -370,3 +371,10 @@ print(water.get_positions())
 ```
 [DP]:https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.120.143001
 [DP-SE]:https://dl.acm.org/doi/10.5555/3327345.3327356
```
Comment (amcadmus) [inline doc/getting-started.md:373]: The section `know limitations` should be placed before the references

File: doc/getting-started.md (hunk 1 above)
Comment (denghuilu) [reply doc/getting-started.md]: solved

File: doc/getting-started.md (hunk 2 above)
Comment (denghuilu) [reply doc/getting-started.md]: solved

**Pull request merged**